testsuite/unit-tests/src/test/java/org/jboss/resteasy/test/util/CaseInsentiveMapTest.java

```diff
@@ -12,13 +12,20 @@
  */
 public class CaseInsentiveMapTest {
 
+   /**
```

Comment:
I think this can be reverted. Other than that I think this looks fine.

Reply:
Done!